tests/integration_tests/performance/test_cpu_template_benchmark.py

```diff
@@ -24,15 +24,15 @@
 
 BASELINES = {
     "Intel": {
-        "deserialize": {"target": 0.025, "delta": 0.02},  # milliseconds
+        "deserialize": {"target": 0.000, "delta": 0.02},  # milliseconds
         "serialize": {"target": 0.015, "delta": 0.02},  # milliseconds
     },
     "AMD": {
-        "deserialize": {"target": 0.0037, "delta": 0.002},  # milliseconds
+        "deserialize": {"target": 0.000, "delta": 0.02},  # milliseconds
         "serialize": {"target": 0.015, "delta": 0.02},  # milliseconds
     },
     "ARM": {
-        "deserialize": {"target": 0.00037, "delta": 0.0006},  # milliseconds
```

Comment:
Would it be more clear to change delta to `"max": 0.02` and below the test would be 

```python
    assert mean < target_max, f"Benchmark result {directory} has changed!"
```

Also, we would need the same for the `serialize` part of the test no?

Reply:
Sure, I'll make the tweak now.